### PR TITLE
stop nuking latest binaries

### PR DESCRIPTION
**Pull request by brthor**
cc @eerhardt @sokket

### CODE REVIEW

File: build_projects/dotnet-cli-build/PublishTargets.cs [hunk 1]
```diff
@@ -73,9 +73,12 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
                 }
                 else
                 {
-                    // This is an old drop of latest so remove all old files to ensure a clean state
+                    Regex versionFileRegex = new Regex(@"(?<version>\d\.\d\.\d)-(?<release>.*)?");
```
Comment (sokket) [inline build_projects/dotnet-cli-build/PublishTargets.cs]: Is there a '.' between 'version' and any \d?

**Comment by sokket** [comment]
LGTM

File: build_projects/dotnet-cli-build/PublishTargets.cs (hunk 1 above)
Comment (brthor) [inline build_projects/dotnet-cli-build/PublishTargets.cs]: I'm not sure I understand.
`<version>` creates a named group, doesn't affect the matching pattern in any way.

Comment (sokket) [inline build_projects/dotnet-cli-build/PublishTargets.cs]: Ah sorry, forgot my Regex for a minute....carry on, nothing to see here...

Comment (brthor) [inline build_projects/dotnet-cli-build/PublishTargets.cs]: :smile:

**Pull request merged**